src/plugins/auto/auto_schedule.cpp

```diff
@@ -166,6 +166,10 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
         _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
             _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
                                              : InferenceEngine::PluginConfigParams::NO;
+        ov::AnyMap option;
```

Comment:
1. If this option variable is necessary?  2. Add test cases to cover both ALLOW_AUTO_BATCH and DEVICE_BIND_BUFFER